dora/core/common/src/main/java/alluxio/underfs/options/GetStatusOptions.java

```diff
@@ -28,15 +28,15 @@ public boolean isIncludeRealContentHash() {
    * @param includeRealContentHash include real content hash flag value
    * @return the updated options object
    */
-  public GetFileStatusOptions setIncludeRealContentHash(boolean includeRealContentHash) {
+  public GetStatusOptions setIncludeRealContentHash(boolean includeRealContentHash) {
     mIncludeRealContentHash = includeRealContentHash;
     return this;
   }
 
   /**
-   * @return the default {@link GetFileStatusOptions}
+   * @return the default {@link GetStatusOptions}
    */
-  public static GetFileStatusOptions defaults() {
-    return new GetFileStatusOptions();
+  public static GetStatusOptions defaults() {
+    return new GetStatusOptions();
```

Comment:
is this change just a file renaming?

Reply:
yes, later on I would try to merge getFileStatus method & getStatus method into one method so I feel like GetStatusOptions is a better name